src/components/InventoryGroups/Modals/AddHostToGroupModal.js

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import validatorTypes from '@data-driven-forms/react-form-renderer/validator-types';
+import componentTypes from '@data-driven-forms/react-form-renderer/component-types';
+import Modal from './Modal';
+import { addHostToGroup } from '../utils/api';
+import apiWithToast from '../utils/apiWithToast';
+import { useDispatch } from 'react-redux';
+import { Button, Text } from '@patternfly/react-core';
+import SearchInput from './SearchInput';
+
+const CreateGroupButton = ({ closeModal }) => (
+    <>
+        <Text>Or</Text>
+        <Button variant="secondary" className="pf-u-w-50" onClick={closeModal}>
+        Create Group
+        </Button>
+    </>
+);
+
+CreateGroupButton.propTypes = {
+    closeModal: PropTypes.func
+};
+
+const createDescription = (deviceIds) => {
+    const systemText =
+      deviceIds.length > 1 ? `${deviceIds?.length} systems` : deviceIds[0].name;
+    return (
+        <Text>
+        Select a group to add <strong>{systemText} </strong> or create a new one.
```

Comment:
According to mocks, we have to show the system display name / id here in `<strong>`

Reply:
Thanks, fixed it